### PR TITLE
Remove initial window/browser size from F94 test

### PR DESCRIPTION
**Pull request by patrickhlauke**
it's irrelevant, as the SC applies at any size, so doesn't need to appear to be limited to any particular one.

This addresses initial concern of https://github.com/w3c/wcag/issues/704 (but that issue has more discussion about the potential need for a normative change for 1.4.4, or addition of note(s) in the understanding, to set some lower bound)

### CODE REVIEW

**Comment by patrickhlauke** [comment]
Any news on this @alastc ?

**Comment by patrickhlauke** [comment]
Expanded slightly to add an extra note

File: techniques/failures/F94.html [hunk 1]
```diff
@@ -45,17 +45,19 @@ <h3>Procedure</h3>
 				<p>In a desktop browser with a resizable window:</p>
 				<ol>
 					<li>Set zoom level to 100%.</li>
-                    <li>Set window size to 1280 pixels wide.</li>
-                    <li>Visit the page to be tested.</li>
-                    <li>Use any of the following methods to resize text when available:
+					<li>Visit the page to be tested.</li>
+					<li>Use any of the following methods to resize text when available:
 						<ul>
 							<li>the zoom feature of the browser</li>
```
Comment (mbgower) [inline techniques/failures/F94.html:51]:
```suggestion
							<li>the zoom feature of the browser,</li>
```

File: techniques/failures/F94.html [hunk 2]
```diff
@@ -45,17 +45,19 @@ <h3>Procedure</h3>
...
+							<li>the zoom feature of the browser,</li>
 							<li>the text-sizing feature of the browser,</li>
 							<li>on-page controls for resizing text.</li>
 						</ul>
 					</li>
-                    <li>Check that the text resizes by one of the methods above, and can be resized to at least 200% of the default.</li>
+					<li>Check that the text resizes by one of the methods above, and can be resized to at least 200% of the default.</li>
 				</ol>
+				<div class="note">
+					<p>If the page has variations that are automatically presented for various screen sizes (e.g. media query breakpoints), you should run the test procedure for each variation.</p>
```
Comment (mbgower) [inline techniques/failures/F94.html:59]:
```suggestion
					<p>If the page has variations that are automatically presented for various screen sizes (for instance, media query breakpoints), you should run the test procedure for each variation.</p>
```

**Review by mbgower — APPROVED** [review]

**Comment by mbgower** [comment]
@patrickhlauke Suggest taking the starting size of the test and incorporating into the meat of the F technique's content. Also if possible would like some guidance on tackling what is reasonable for lower resolutions. @scottaohara would be a good person to run your draft text by

**Comment by bruce-usab** [comment]
@scottaohara requests that WCAG2ICT TF gives this issue some attention.

**Comment by scottaohara** [comment]
thanks to @dbjorge for calling out #2101 as a directly related issue for a minimum testing resolution/viewport.

**Comment by bruce-usab** [comment]
On call 12/8, @dbjorge also notes intersection with #1522 #730 and https://github.com/w3c/wcag2ict/issues/4

**Comment by patrickhlauke** [comment]
this one *may* actually be superseded by https://github.com/w3c/wcag/pull/3739 - need to double-check once/if that one is merged if this is still needed. certainly the note

```
<div class="note">
	<p>If the page has variations that are automatically presented for various screen sizes (for instance, media query breakpoints), you should run the test procedure for each variation.</p>
</div>
```

appears new here compared to #3739 so this may be a refocused/separate PR just about the note

**Comment by patrickhlauke** [comment]
As F94 uses `vw` as font size, it's irrelevant to point out in the note that this needs to be repeated at different sizes. closing

**Pull request closed without merging**